core/build/build.odin

```diff
@@ -1,21 +1,49 @@
 package build
 
 import "core:strings"
-import "core:fmt"
 import "core:os"
-import "core:c/libc"
 import "core:path/filepath"
 import "core:sync"
 import "core:encoding/json"
+import "core:log"
+import "core:runtime"
+import "core:fmt"
+
+import "core:c/libc" // TODO: remove dependency on libc after core:os2
+
+
+@init
+_ :: proc() {
+	_context = create_default_context()
+}
+
+create_default_context :: proc() -> runtime.Context {
+	context = runtime.default_context()
+	context.allocator = context.temp_allocator
+	context.logger = log.create_console_logger(.Debug, {.Level})
+	return context
+}
+
+default_context :: proc() -> runtime.Context {
```

Comment:
I'm not sure what's the roundabout with creating a global context with a logger, to then have the user retrieve that context. Can't `default_context` create the context and return newly-created instead of returning a global context? I'm not seeing the purpose here...

Reply:
i can move the global into a `@static` in `default_context`, otherwise `create_console_logger` would keep creating a new one on each call.

Reply:
![image](https://github.com/odin-lang/Odin/assets/31741257/926e59b9-19b5-4049-99a0-17c84d3ed7c2)
this don't work